msteams-platform/resources/samesite-cookie-update.md

```diff
@@ -80,6 +80,43 @@ Android WebView is a Chrome system component that allows Android apps to display
 > * You must declare first party cookies as `SameSite=Lax` or `SameSite=Strict`, as appropriate.
 > * You must declare third party cookies as `SameSite=None; Secure`.
 
+## Third party cookies deprecation
+
+All third party cookies set in the top-level domain are blocked when that domain is embedded in an iframe. This third party cookie deprecation is starting to roll out for all major browsers, with Chrome currently at a 1% rollout and other major browsers like Firefox and Safari to follow.
```

Comment:
Let's rephrase in a way that avoids reporting on rollout status. It's risky (and unadvised) enough to attempt reporting on our own MS rollouts, and even more so on external products. Maybe something like this:

Third-party cookies are in the process of being deprecated across all major browsers. As that happens, all third-party cookies set in the top-level domain will be blocked when that domain is embedded in an `iframe`.

Reply:
Updated as suggested.